### PR TITLE
Fix isVisible false negative on rounded-cornered elements

### PR DESCRIPTION
**Pull request by mrmacete**
- test also the center

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -285,13 +285,18 @@ IS_ELEMENT_VISIBLE_SCRIPT = `function isElementVisible(params) {
     return document.elementFromPoint(x, y)
   };
 
-  // Return true if any of its four corners are visible
+  // Return true if any of its four corners or the
+  // center are visible
```
Comment (oleavr) [inline index.js]: How about replacing the comment with something like:

```js
var anyOfFourCornersOrCenterVisible = (... || ... || ...);`

return {
  visible: anyOfFourCornersOrCenterVisible
};
```

Comment (oleavr) [inline index.js]: Or, possibly clearer:

```js
var anyOfFourCornersVisible = ...;
var centerVisible = ...;

return {
  visible: anyOfFourCornersVisible || centerVisible
};
```

**Comment by mrmacete** [comment]
totally agree!

**Comment by oleavr** [comment]
👍

**Comment by oleavr** [comment]
+ frida-uiwebview@1.1.4

**Pull request merged**